cranelift/codegen/src/isa/x86/abi.rs

```diff
@@ -394,21 +397,60 @@ fn callee_saved_gprs(isa: &dyn TargetIsa, call_conv: CallConv) -> &'static [RU]
     }
 }
 
+/// Get the set of callee-saved floating-point (SIMD) registers.
+fn callee_saved_fprs(isa: &dyn TargetIsa, call_conv: CallConv) -> &'static [RU] {
+    match isa.triple().pointer_width().unwrap() {
+        PointerWidth::U16 => panic!(),
+        PointerWidth::U32 => &[],
+        PointerWidth::U64 => {
+            if call_conv.extends_windows_fastcall() {
+                // "registers RBX, ... , and XMM6-15 are considered nonvolatile and must be saved
+                //  and restored by a function that uses them."
+                // as per https://docs.microsoft.com/en-us/cpp/build/x64-calling-convention as of
+                // 02-05-2020.
```

Comment:
nit: can you expand the date as `February 2nd, 2020`, please? I read this as "May 2nd", as a European person :)

Reply:
Oops, yeah I should expand that :)

Reply:
Or use `2020-02-05`. ([ISO 8601](https://en.wikipedia.org/wiki/ISO_8601] format)